Makefile-os

```diff
@@ -101,7 +99,7 @@ initialize_docker: create_env_file build_docker_image
 # would be uid 9500 regardless of host at this point (this is only fixed when
 # the container is up, through the command defined in docker-compose.yml),
 # which is wrong for local development.
-	docker compose run --rm --user ${UID}:${GID} web make update_deps
+	docker compose run --rm web make update_deps
```

Comment:
The comment above explains what this isn't true: the script is not an actual entrypoint, just a default command we run on local envs. It's bypassed when using `docker compose run` with a custom command like we are here. So that would cause `make initialize_docker` to run that initial `make update_deps` with the wrong UID/GID...

Reply:
It is in fact an entrypoint now. https://github.com/mozilla/addons-server/pull/22206/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R36

I guess the comment is now redundant or outdated at least..

Reply:
Ah, I missed that. Yeah the comment needs to be updated then. I'll test locally now that the other PR has been approved and that one rebased.